collectors/python.d.plugin/zscores/README.md

```diff
@@ -0,0 +1,146 @@
+<!--
+---
+title: "zscores"
+description: "Use statistical anomaly detection to narrow your focus and shorten root cause analysis."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/zscores/README.md
+---
+-->
+
+# Z-Scores - basic anomaly detection for your key metrics and charts
+
+Smoothed, rolling [Z-Scores](https://en.wikipedia.org/wiki/Standard_score) for selected metrics or charts.
+
+This collector uses the [Netdata rest api](https://learn.netdata.cloud/docs/agent/web/api) to get the `mean` and `sigma`
```

Comment:
I will give a suggestion, please, instead `sigma`, let us define as standard deviation and give indication that it is represented as sigma, because we are calculating the square root of the second moment, not the sigma.

Reply:
agree - clearer to use `stddev` instead of `sigma` as will be clearer to people.